scripts/src/lib/template/templates/git/workflow.yml

```diff
@@ -18,16 +18,17 @@ jobs:
     steps:
       - name: checkout repository
         uses: actions/checkout@v4
-      - name: validate gradle wrapper
-        uses: gradle/wrapper-validation-action@v2
       - name: setup jdk ${{ matrix.java }}
         uses: actions/setup-java@v4
         with:
           java-version: ${{ matrix.java }}
           distribution: 'microsoft'
-      - name: make gradle wrapper executable
-        run: chmod +x ./gradlew
-      - name: build
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
```

Comment:
No, sorry I dislike how this caches stuff by default, its soo easy to accidently cache private information in a way that can be accessed publicly, I don't want to be part of it.

Reply:
couldnt you just use `cache-disabled: true` to disable that though